### PR TITLE
[IMP] test_pylint: Support new enviroment variable EXCLUDE_LINT

### PR DESCRIPTION
**Pull request by JesusZapata**
Fix https://github.com/Vauxoo/maintainer-quality-tools/issues/182

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
This PR add  `EXCLUDE_LINT` environment variable used by test_pylint script

File: travis/test_pylint [hunk 1]
```diff
@@ -136,9 +136,13 @@ pylint_rcfile = os.path.join(
     'cfg',
     "travis_run_pylint.cfg")
 modules_cmd = []
-if os.environ.get('INCLUDE_LINT'):
-    for path in os.environ.get('INCLUDE_LINT').split(' '):
-        modules_cmd.extend(['--path', path])
+if os.environ.get('INCLUDE_LINT') or os.environ.get('EXCLUDE_LINT'):
+    if os.environ.get('INCLUDE_LINT'):
```
Comment (moylop260) [inline travis/test_pylint]: You have this `if` 2 times?

Comment (moylop260) [inline travis/test_pylint]: In fact, we could avoid the `if` using:
`for path in (os.environ.get('INCLUDE_LINT') or '').split(' ')`

Comment (JesusZapata) [inline travis/test_pylint]: @moylop260 
Refactor the code using lambda function and avoid using if

**Comment by moylop260** [comment]
Could you add a line in the .travis.yml with EXCLUDE_LINT and other one with INCLUDE_LINT to test it?

**Comment by JesusZapata** [comment]
@moylop260 
I add one line in .travis.yml for test of `EXCLUDE_LINT` but by attaching the `INCLUDE_LINT` the test 

image

Can we close this PR and in another review because the environment variable INCLUDE_LINT is failing?

**Comment by moylop260** [comment]
Currently works:
 - normal case:
   - pylint GLOBAL_PATH

 - exclude case:
   -  [pylint path for path in GLOBAL_PATH if not is_excluded]

Then the fix should be:

- include case:
   - [pylint path for path in GLOBAL_PATH if not is_excluded or is_included]

I mean, avoid use the global path for include cases.

**Comment by JesusZapata** [comment]
@moylop260 
Thanks for you help!

File: .travis.yml [hunk 2]
```diff
@@ -41,6 +41,8 @@ env:
   - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="89" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
   - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="82" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
   - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false"  # To check VERSION empty or unset case.
+  - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="15" TRAVIS_PULL_REQUEST="false" EXCLUDE_LINT="$HOME/build/$TRAVIS_REPO_SLUG/broken_lint"  # To check EXCLUDE_LINT
+  - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false" INCLUDE_LINT="$HOME/build/$TRAVIS_REPO_SLUG"  # To check INCLUDE_LINT
```
Comment (moylop260) [inline .travis.yml]: Use just a submodule in order to check that is testing just that module

Comment (JesusZapata) [inline .travis.yml]: @moylop260 
I add `INCLUDE="test_module"` into .travis.yml 
Is what you mean?

Comment (JesusZapata) [inline .travis.yml]: @moylop260 
I add `INCLUDE="test_module"` into .travis.yml 
This is what you meant?

**Pull request merged**